data/lua/connector_pkmn_rb.lua

```diff
@@ -60,16 +60,18 @@ function generateLocationsChecked()
 	dexsanity = uRange(DexSanityAddress, 19)
 	rod = u8(RodAddress)
 
-	data = {}
+	data = {rod}
```

Comment:
The rod data needs to be after the hidden items data. The order of the data in this table needs to remain the same.